metadata_json_deps.gemspec

```diff
@@ -14,11 +14,10 @@ Gem::Specification.new do |s|
   s.executables << 'generate-fixtures-yaml'
   s.executables << 'metadata-json-deps'
 
-  # puppet_forge requires Ruby 2.4
-  s.required_ruby_version = Gem::Requirement.new('>= 2.4.0')
+  s.required_ruby_version = Gem::Requirement.new('>= 2.7.0')
 
   s.add_runtime_dependency 'puppet_forge', '>= 2.2', '< 5'
-  s.add_runtime_dependency 'puppet_metadata', '>= 0.3.0', '< 4'
+  s.add_runtime_dependency 'puppet_metadata', '~> 3.0'
```

Comment:
Any reason you want version 3 specifically?

Reply:
Version 3 requires Ruby 2.7. I want to bump the minimal ruby version for metadata_json_deps to 2.7 so this is compatible. I don't think it's required to bump to 3 here, but I also think it doesn't hurt.